src/sub-blocks/PriceCard/PriceCard.scss

```diff
@@ -63,8 +63,10 @@ $block: '.#{$ns}price-card';
     }
 
     &__title {
-        @include text-size(subheader-3);
-        margin-bottom: $indentSM;
+        @include add-specificity(&) {
```

Comment:
I see more `specificity` in this pr why they is appear ?)

Reply:
Because, React.createElement works with classes slightly different. And styles of yfm override styles from component classes